src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/configuration/KafkaConsumerConfig.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.companieshouse.chipsrestinterfacesconsumer.configuration;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.annotation.EnableKafka;
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+import uk.gov.companieshouse.chips.ChipsRestInterfacesSend;
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.avro.AvroDeserializer;
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.common.ApplicationLogger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@EnableKafka
+@Configuration
+public class KafkaConsumerConfig {
+
+    @Value("${kafka.broker.address}")
+    private String brokerAddress;
+
+    @Value("${RETRY_THROTTLE_RATE_SECONDS}")
+    private long retryThrottleSeconds;
+
+    @Value("${MAX_RETRY_ATTEMPTS}")
+    private int maxRetryAttempts;
+
+    @Autowired
+    private ApplicationLogger logger;
+
+    @Bean
+    public ConsumerFactory<String, ChipsRestInterfacesSend> consumerFactory() {
+        Map<String, Object> props = new HashMap<>();
+        props.put(
+                ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG,
```

Comment:
i think we'll need to specify  enable.auto.commit = false. Otherwise, i'll double check but I think auto commit will only write the commit offsets once all the messages returned via poll are completed